sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/LiveMetrics/DataCollection/DoubleBuffer.cs

```diff
@@ -14,17 +15,17 @@ namespace Azure.Monitor.OpenTelemetry.AspNetCore.LiveMetrics.DataCollection
     /// </summary>
     internal class DoubleBuffer
     {
-        private DocumentBuffer _currentBuffer = new();
+        private ConcurrentQueue<DocumentIngress> _currentBuffer = new();
```

Comment:
What is the perf gain with this change?

Reply:
This line isn't associated with any perf change.

The DocumentBuffer class wrapped the ConcurrentQueue and managed the capacity. After removing the max capacity, this class didn't serve any purpose so I removed it.